packages/ui/src/i18n/dictionaries/authenticator/defaultTexts.ts

```diff
@@ -21,7 +24,18 @@ export const defaultTexts = {
   CREATE_ACCOUNT: 'Create Account',
   CREATING_ACCOUNT: 'Creating Account',
   EMAIL_ADDRESS: 'Email',
+  ENTER_BIRTHDATE: 'Enter your Birthdate',
```

Comment:
Feels weird that we duplicate this text, I wonder if we should just pull from the `en` dictionary?

Reply:
I feel the same. Or maybe vice verse :)